plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,31 @@ struct MutantTimeoutCtxTbl {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("st_id")
+struct MarkedMutantTbl {
+    @ColumnName("st_id")
+    long mutationStatusId;
+
+    long id;
+
+    uint line;
+
+    uint column;
+
+    string path;
+
+    ulong to_status;
+
+    SysTime time;
+
+    string rationale;
+
+    string text;
```

Comment:
I recommend changing the variable name to reflect what it is.
Is it the original text? Or the mutated.
I actually do not remember from our conversation so I would have to read how it is used.